README.md

```diff
@@ -204,7 +204,7 @@ end
 Now load the ABI and pass the method signature. Note that the address needs to be converted to bytes:
 
 ```elixir
-address           = "0x123" |> String.slice(2..-1) |> Base.decode16(case: :mixed)
+address           = "0x123" |> String.slice(2..-1) |> Base.decode16!(case: :mixed)
```

Comment:
can you please use a real world address string. I get an error using this example

```elixir
"0x123" |> String.slice(2..-1) |> Base.decode16!(case: :mixed)

** (ArgumentError) odd-length string
    (elixir 1.13.1) lib/base.ex:352: Base.decode16!/2
```

Reply:
Yes, I can take advantage and change that too.

Reply:
Done, please try the examples now.